docs/memcached.md

```diff
@@ -0,0 +1,64 @@
+# `memcached` Metrics Receiver
+
+The memcached receiver can retrieve stats from your memcached server through the . 
+
+
+## Configuration
+
+Following the guide for [Configuring the Ops Agent](https://cloud.google.com/stackdriver/docs/solutions/agents/ops-agent/configuration#file-location), add the required elements for your memcached configuration.
+
+To configure a receiver for your memcached metrics, specify the following fields:
+
+| Field                 | Default                   | Description |
+| ---                   | ---                       | ---         |
+| `type`                | required                  | Must be `memcached`. |
+| `endpoint`            | `localhost:3306`          | The url, or unix socket file path, for your memcached server. |
+| `collection_interval` | `60s`                     | A [time.Duration](https://pkg.go.dev/time#ParseDuration) value, such as `30s` or `5m`. |
+
+Example Configuration:
+
+```yaml
+metrics:
+  receivers:
+    redis_metrics:
+      type: memcached
+      address: localhost:3306
```

Comment:
Is this field `endpoint` or `address`? Line 15 says `endpoint`. 

BTW, can we keep the example configuration with minimum customization, as users tend to copy paste it. We'd probably prefer them sticking with the default `60s` interval. In other words, we can remove these two lines?

```
address: localhost:3306
collection_interval: 30s
```

Reply:
Updated, however I'm pretty sure we require that collection interval is set as it is set to required in[ the validation](https://github.com/GoogleCloudPlatform/ops-agent/blob/d1c729ef424c1848b642838d5c32a16988290114/confgenerator/config.go#L375).  but updated to 60s and removed `address/endpoint` field in example. it is endpoint btw.

Reply:
You are right. I just realized today that the `collection_interval` is a required field in the implementation (It's supposed to be optional). We'll fix that separately as it's not directly related to this PR.

Resolving this thread.